package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "github-info",
+  "version": "1.0.0",
+  "main": "index.ts",
+  "license": "MIT",
+  "scripts": {
+    "start": "npx ts-node src/index.ts",
+    "lint": "npx eslint src",
+    "database-up": "docker compose -f infra/docker-compose.yaml up",
```

Comment:
Hello Gonçalo,

First of all, thank you for the test!

I'm David, a member of LovelyStay's backend development team :)

----

Shouldn't this `database-up` script have a `-d` flag so it spins up the container in detached mode? Otherwise we're stuck with a shell showing the container logs

Reply:
I personally prefer having access to the logs directly in the shell when debugging and opening a different shell to run (especially since you can just pass the flag manually), but I do agree that this would be needed for a final deployment script